docker/crashpad_symbolize/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM cobalt-base
+
+RUN apt update -qqy \
+    && apt install -qqy --no-install-recommends jq build-essential zlib1g-dev libzstd-dev \
```

Comment:
style nit: i'd format it like so:

```
&& apt install -qqy --no-install-recommends \
  jq build-essential zlib1g-dev libzstd-dev \
```

Less churn later when the dependencies inevitably end up changing down the road.